src/parcera/core.cljc

```diff
@@ -135,7 +136,7 @@
         | 'tab'
         | 'formfeed'
         | 'backspace'
-        | #'\\P{M}\\p{M}*+'; (* https://www.regular-expressions.info/unicode.html *)")
+        | #'[^\\u0300-\\u036F\\u1DC0-\\u1DFF\\u20D0-\\u20FF][\\u0300-\\u036F\\u1DC0-\\u1DFF\\u20D0-\\u20FF]*'; (* https://www.regular-expressions.info/unicode.html *)")
```

Comment:
could you please update the comment ? Maybe put which kind of unicode characters is this suppose to match. I am quite clueless about this regex ... not that I fully understood the previous one anyway 😅

Reply:
I updated the doc string some. :smile: